core/kazoo_voicemail/doc/README.md

```diff
@@ -25,14 +25,27 @@ All media recordings (private_media) remain in account db to be moved to message
 Kazoo datamanager handle of fetching the actual media attachment from account db anytime MODB documents attachemnt tries to be reached. This process is seamless from users and are handled entirely by
 Kazoo data layer.
 
-These are configurations (`system_config/callflow/voicemail`) for controlling migration process:
+## System Config settings
```

Comment:
please revert this, it's updated in https://github.com/2600hz/kazoo/pull/4106

Reply:
@icehess i was here first ;) but okay